File: jdroid-android-google-inappbilling/src/main/java/com/jdroid/android/google/inappbilling/client/ItemType.java
package com.jdroid.android.google.inappbilling.client;

import com.android.billingclient.api.BillingClient;

public enum ItemType {
	
	MANAGED("inapp"),
	SUBSCRIPTION("subs");
	
	private String type;
	
	ItemType(String type) {
		this.type = type;
	}
	
	@BillingClient.SkuType
	public String getType() {
		return type;
	}
}

File: jdroid-android-google-inappbilling/src/main/java/com/jdroid/android/google/inappbilling/client/ItemType.kt
package com.jdroid.android.google.inappbilling.client

import com.android.billingclient.api.BillingClient

enum class ItemType constructor(
    @BillingClient.SkuType
    val type: String
) {

    MANAGED("inapp"),
    SUBSCRIPTION("subs")
}
